### PR TITLE
Add delete message on chat

### PR DESCRIPTION
**Pull request by JeanColombel93**
image

### CODE REVIEW

File: app/controllers/messages_controller.rb [hunk 1]
```diff
@@ -12,6 +12,12 @@ def create
     end
   end
 
+  def destroy
+    @message = Message.find(params[:id])
+    @message.destroy
+    redirect_to communities_path
```
Comment (annemano) [inline app/controllers/messages_controller.rb:18]: shouldn't it redirect to the community itself where we were?

File: app/views/messages/_message.html.erb [hunk 2]
```diff
@@ -5,6 +5,11 @@
       <small><%= message.created_at.strftime("%a %b %e at %l:%M%p") %></small>
     </i>
     <p><%= message.content %></p>
+    <% if message.user == current_user %>
+      <%= link_to community_message_path(message), method: :delete, data: { confirm: 'Delete this message?' } do %>
```
Comment (annemano) [inline app/views/messages/_message.html.erb:9]: add remote:true

File: app/views/messages/_message.html.erb [hunk 3]
```diff
@@ -13,5 +18,10 @@
       <small><%= message.created_at.strftime("%a %b %e at %l:%M%p") %></small>
     </i>
     <p><%= message.actiontext_content %></p>
+    <% if message.user == current_user %>
+      <%= link_to community_message_path(message), method: :delete, data: { confirm: 'Delete this message?' } do %>
```
Comment (annemano) [inline app/views/messages/_message.html.erb:22]: add remote: true

**Comment by JeanColombel93** [comment]
Made the changes. Thank you !

**Pull request merged**